docs/libcurl/opts/CURLSHOPT_SSL_SESSIONID_CACHE_SIZE.3

```diff
@@ -0,0 +1,60 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 1998 - 2022, Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at https://curl.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" **************************************************************************
+.TH CURLSHOPT_SSL_SESSIONID_CACHE_SIZE.3 3 "24 May 2022" "libcurl 7.84.0" "libcurl Manual"
+.SH NAME
+CURLSHOPT_SSL_SESSIONID_CACHE_SIZE - set the size of the SSL session-ID cache
+.SH SYNOPSIS
+.nf
+#include <curl/curl.h>
+
+CURLSHcode curl_share_setopt(CURLSH *share, CURLSHOPT_SSL_SESSIONID_CACHE_SIZE,
+                             long size);
+
+.fi
+.SH DESCRIPTION
+Set the size of the SSL session-ID cache. The size can only be set before the
+SSL session-ID cache of a share handle is enabled by calling
+curl_share_setopt(share, CURLSHOPT_SHARE, CURL_LOCK_DATA_SSL_SESSION).
+.SH DEFAULT
+8
```

Comment:
I believe this is 8 for shared handles, but 5 by default for an easy handle.

Reply:
That's correct. The default value 5 for an easy handle is on the corresponding manual page `CURLOPT_SSL_SESSIONID_CACHE_SIZE.3`.